weblate/trans/views/search.py

```diff
@@ -154,6 +154,11 @@ def search(request, path=None):
         units = get_paginator(
             request, units.order_by_request(search_form.cleaned_data, obj)
         )
+
+        # Calculate total number of strings and words
+        total_strings = units.paginator.count
+        total_words = sum(unit.num_words for unit in units.object_list)
```

Comment:
The strings number is for all strings, but words number just for the current page.